build.sbt

```diff
@@ -550,11 +550,22 @@ lazy val `parser-service` = (project in file("lib/scala/parser-service"))
 lazy val `docs-generator` = (project in file("lib/scala/docs-generator"))
   .dependsOn(syntax.jvm)
   .dependsOn(cli)
+  .configs(Benchmark)
   .settings(
     libraryDependencies ++= Seq(
       "commons-cli" % "commons-cli" % commonsCliVersion
     ),
-    mainClass := Some("org.enso.docs.generator.Main")
+    mainClass := Some("org.enso.docs.generator.Main"),
```

Comment:
Correct, there should be no main I think?

Reply:
No, this is not the module Marcin referred. Previously there was also a `docs-runner` module that is now integrated with `enigine-runner`. This on the other hand is a bigger module that was merged some while ago, doing what the previous' python app did, and also opening up a interface for doc parser to be called from wherever you want to - in our case - from the compiler's pass. Hence to repeat, that's not that module :))

Reply:
But is this module ever called as a separate Java application? Because that is, I think, the only situation where it would need to have the main class specified.

Reply:
Yes - it is. If you call docs generator project from sbt it'll take all the standard library files, munch them and prepare docs for the website.

Reply:
Ah I see.

But why isn't this ran through the engine-runner instead? I don't see why we'd want to have multiple entry-points for this.

Reply:
+1 to Radeks doubts. You have just created a whole docs generator but are not using it in the only place that's generating docs? Oo